Fix uninitialised values

DamMicSzm opened the pull request:
> !!!!!!!!!! Please delete the instructions below and replace with PR description
>
> If you have an issue number, please use a syntax of
> `Fixes #12345` and a brief change description
>
> If you do not have an issue number, please have a good description of
> the problem and the fix. Help the reviewer understand what to expect.
>
> Make sure you delete these instructions (to prove you have read them).
>
> !!!!!!!!!! Instructions end

Commit:
Fix uninitialised value in KeySet

Commit:
cleanup

Commit:
Fix uninitilised mGlobalAttributeEndIndex

arkq commented on src/app/clusters/operational-credentials-server/operational-credentials-server.cpp:
```diff
@@ -599,11 +599,11 @@ bool emberAfOperationalCredentialsClusterAddNOCCallback(app::CommandHandler * co
-    auto & fabricTable               = Server::GetInstance().GetFabricTable();
+    CHIP_ERROR err                                = CHIP_NO_ERROR;
+    FabricIndex newFabricIndex                    = kUndefinedFabricIndex;
+    Credentials::GroupDataProvider::KeySet keyset = {};
```
How about setting default values in the class itself?

DamMicSzm replied:
It may be as you suggest

Commit:
Review fix

Damian-Nordic committed:
[dnssd] remove constants for _A subtype and AP TXT field (#28332)

* [dnssd] remove constants for _A subtype and AP TXT field

The _A subtype and AP TXT field were removed from the spec
long time ago but the constants describing the maximum
lengths are still present in the SDK.

* Restyled by clang-format

---------

Co-authored-by: Restyled.io <commits@restyled.io>
